Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -348,7 +348,7 @@ Describe "BqJob-Extract-Load" {
 
     AfterAll {
         $test_set | Remove-BqDataset -Force
-        $bucket.Name | Remove-GcsBucket -Force
+        Remove-GcsBucket "ps_test_$r" -Force
```

Comment:
This seems fragile and could break if someone changes the bucket name on the creation side. Why not use `$bucket.Name` or a new variable `$bucketName`?